src/ProgressOnderwijsUtils/DistinctArray.cs

```diff
@@ -59,7 +59,7 @@ public static DistinctArray<T> FromDistinctNonMutatedArray([NotNull] T[] items,
         public static DistinctArray<T> FromPossiblyNotDistinct([NotNull] IEnumerable<T> items, IEqualityComparer<T> comparer)
             => new DistinctArray<T>(new HashSet<T>(items, comparer).ToArray());
 
-        readonly T[] items;
+        readonly T[]? items;

```

Comment:
Dit klopt toch niet? Wanneer kan deze nul zijn?

Reply:
Het is een struct; het moet nullable zijn.